amy/dashboard/views.py

```diff
@@ -229,7 +234,33 @@ def training_progress(request):
     progress_welcome = get_passed_or_last_progress(request.user, "Welcome Session")
     progress_demo = get_passed_or_last_progress(request.user, "Demo")
 
-    if request.method == "POST":
+    context = {
+        "title": "Your training progress",
+        "progress_training": progress_training,
+        "progress_get_involved": progress_get_involved,
+        "progress_welcome": progress_welcome,
+        "progress_demo": progress_demo,
+    }
+    return render(request, "dashboard/training_progress.html", context)
+
+
+class GetInvolvedCreateView(LoginRequiredMixin, RedirectSupportMixin, AMYCreateView):
```

Comment:
I think `RedirectSupportMixin` is not needed here, as `ModelFormMixin` (part of Django's `CreateView`) handles redirect after form submission.